content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -240,7 +243,7 @@ To enable *eLearning* courses on a *Helpdesk* team, go to :menuselection:`Helpde
 --> Teams` and select a team, or create a :doc:`new one
```

Comment:
```suggestion
--> Helpdesk Teams` and select a team, or create a :doc:`new one
```

I'd just search to make sure al instances of the config menu has "Helpdesk Teams"